znzlib/CMakeLists.txt

```diff
@@ -1,35 +1,34 @@
 #File Generated by Hans J. Johnson
 #Please contact hans-johnson@uiowa.edu for making enhancments/corrections
-set(ZNZLIB_SRC znzlib.c)
-include_regular_expression("^.*$")
+cmake_minimum_required(VERSION 3.11 FATAL_ERROR)
```

Comment:
minimum required version is only needed at the topmost level.

Reply:
if you do not intend to build zlib separately, it would not be necessary here.

Reply:
znz sometimes uses zlib but AFNI doesn't build it so the cmake find module is fine for linking against it.

Some of the code in the cmake files of the sub directories was to enable individual builds of those sub-directories. I shall abandon that for the sake of being concise